spec/features/collection_drafts/forms/spatial_information_form_spec.rb

```diff
@@ -545,9 +535,8 @@
     it 'populates the form with the values including orbital spatial data' do
       # Spatial Extent
       within '.spatial-extent' do
-        expect(page).to have_no_checked_field('Horizontal')
-        expect(page).to have_no_checked_field('Vertical')
-        expect(page).to have_checked_field('Orbit')
+        expect(page).to have_field('Spatial Coverage Type', with: 'ORBITAL')
+
         expect(page).to have_field('Swath Width', with: '1.0')
         expect(page).to have_field('Period', with: '2.0')
         expect(page).to have_field('Inclination Angle', with: '3.0')
```

Comment:
I know this might be tedious (sorry), but I feel like we should add tests for the new options for Spatial Coverage Type (that they said folks likely wouldn't use) - `Horizontal and Vertical` and `Orbital and Vertical`